options.go

```diff
@@ -991,7 +991,7 @@ func (opts *Options) SetFIFOCompactionOptions(value *FIFOCompactionOptions) {
 // GetStatisticsString returns the statistics as a string.
 func (opts *Options) GetStatisticsString() string {
 	sString := C.rocksdb_options_statistics_get_string(opts.c)
-	defer C.free(unsafe.Pointer(sString))
+	defer C.rocksdb_free(unsafe.Pointer(sString))
```

Comment:
Confirmed: this is allocated by strsdup and should be freed as well: https://github.com/facebook/rocksdb/blob/6ec6a4a9a49e506eff76aebd104d30be6a2d36cc/db/c.cc#L2706

Reply:
alloc by rocksdblib then should released by rocksdb_free